solvexia_sdk/datasteps/datasteps.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+import requests
+import json
+import sys
+from solvexia_sdk import api
+
+class datasteps:
+    def __init__(self, stepId):
+        self.stepId = stepId
+
+    def getDataStep(self):
+        response = api.apiGet(f"steps/{self.stepId}")
+        api.statusCodeCheck(response, "Error getting data step")
+        return response.json()
+
+    def getDataStepProperties(self):
```

Comment:
If you have `getProcessList` then should be `getDataStepPropertyList`. Or the other way around.

Reply:
fixed